tests/test_app.py

```diff
@@ -56,7 +56,13 @@ def test_model_classify_no_image_file(client):
 
 
 def test_model_classify_invalid_image_url(client):
-    response = client.get('/classify', query_string={'image': 'https://example.com/not-an-image.jpg'})
+    response = client.get('/classify', query_string={'image': 'https://not-a-url/image.jpg'})
+    assert response.status_code == 200
+    assert response.json == {'error': 'unable to resolve image url'}
+
+
+def test_model_classify_non_existent_image_url(client):
+    response = client.get('/classify', query_string={'image': 'https://example.com/non-existent-image.jpg'})
```

Comment:
I'd ideally like to avoid hitting a domain that isn't ours in tests, it's a bit mean to the site operators...

Reply:
It's such a common use of example.com, I doubt it's any trouble to them, in that I doubt it's actually looking for any resources on request.

We could hit one or our GS domains with a non-existent path but then we're doing the same to Google on an endpoint that actually does need to look for the requested resource.

Reply:
Not a big deal, it's fine